packages/cardano-services/src/StakePool/DbSyncStakePoolProvider/DbSyncStakePoolProvider.ts

```diff
@@ -107,6 +126,9 @@ export class DbSyncStakePoolProvider extends DbSyncProvider(RunnableModule) impl
         return (options?: QueryStakePoolsArgs) =>
           this.#builder.queryPoolMetrics(hashesIds, totalStake, useBlockfrost, options);
       case 'apy':
+        if (this.#responseConfig?.search?.metrics?.apy === false) {
```

Comment:
```suggestion
        if (this.#responseConfig?.search?.metrics?.apy) {
```
If there's a reason for explicit check then there's a bug [here](https://github.com/input-output-hk/cardano-js-sdk/pull/641/files#diff-3a938d2c168d4b251612c6efabf201ac70e7585c3c1c2149b3c34e71981e6666R273)

Reply:
6a9b545095363a8468e81f7f5e5a606c148cc86b